### ISSUE TITLE
扩展库规划

### ISSUE THREAD

**Issue by songxiaosheng** [apache/dubbo-spi-extensions#232]
# dubbo-spi-extensions

**扩展库说明**：
Dubbo以微内核的架构开发核心的功能包含在Dubbo主参考中 [Dubbo](https://github.com/apache/dubbo)  扩展库则是微内核架构下的对Dubbo生态的扩展来兼容不同公司不同的微服务场景，完善的生态可有有效的提升用户体验，降低用于使用成本，下面主要从两个方面进行展开完善：
- 输入：通过与Dubbo主仓库进行关联，接入自动化CI，完善开发规范，发布规范等。
- 输出：通过解决更多用户场景、文档完善、扩展市场等方式来为用户和社区创造价值与反馈。

**持续开发计划**：

- [ ] 添加新功能
- [ ] 解决问题
- [ ] dubbo核心ci集成（遵循第二级大版本与主仓库保持一致，小版本各自维护）
- [ ] 兼容dubbo3.x版本

  
**代码审查规范**：
- [ ]  代码审查规范，确保代码质量和一致性。

**社区管理**：
- [ ] 定期小会同步

**文档维护**：
- [ ] **代码文档（设计手册）**：编写清晰的代码注释和文档，让其他开发者能够理解代码。
- [ ] **用户文档**（官网扩展市场）：提供易于理解的用户手册和文档，帮助新用户快速上手和使用项目。

仓库文档任务如下：
- [ ] 完善README.md 
    - 参考：https://github.com/dapr/components-contrib
- [ ]  完善行为准则.md
   - 参考： https://github.com/dapr/community/blob/master/CODE-OF-CONDUCT.md
- [ ]   完善用户列表.md 
  - 参考：https://github.com/dapr/community/blob/master/ADOPTERS.md
- [ ]   完善发布流程.md 参考：https://github.com/dapr/community/blob/master/release-process.md

**Comment by aofall** [comment]
建议 `兼容dubbo3.x版本` 版本这个计划再细分一下，3.0、3.1、3.2 版本需要逐步适配，未来还有3.3版本

**Comment by songxiaosheng** [comment]
> 建议 `兼容dubbo3.x版本` 版本这个计划再细分一下，3.0、3.1、3.2 版本需要逐步适配，未来还有3.3版本

文档已更新，求认领😃

**Comment by aofall** [comment]
> 文档已更新，求认领😃

我先认领 `兼容dubbo 3.1.x版本` 的任务

**Comment by songxiaosheng** [comment]
Cloud you add my DingTalk account? No, let's discuss the solution together. Thank you
![image](https://github.com/apache/dubbo-spi-extensions/assets/81170548/ac281d3f-d037-4d8b-84c8-4764f699e802)

**Comment by finefuture** [comment]
Can I participate in the development of version 3.2?

**Comment by songxiaosheng** [comment]
> Can I participate in the development of version 3.2?

yes,you can add  my dingding id then help fix some  components,thanks

**Comment by finefuture** [comment]
> > Can I participate in the development of version 3.2?
> 
> yes,you can add my dingding id then help fix some components,thanks

Okay, I'll add you later.

**Comment by finefuture** [comment]
I want to participate in these three modules:
- dubco-cluster-specify -address-common
- dubbo-cluster-specify-address-dubbo2
- dubbo-cluster-specify-address-dubbo3

**Comment by songxiaosheng** [comment]
> I want to participate in these three modules:
> 
> * dubco-cluster-specify -address-common
> * dubbo-cluster-specify-address-dubbo2
> * dubbo-cluster-specify-address-dubbo3

done

**Comment by chuntaojun** [comment]
polaris 相关的都 assign 给我吧 @chuntaojun

**Comment by songxiaosheng** [comment]
> polaris 相关的都 assign 给我吧 @chuntaojun

has updated

**Comment by finefuture** [comment]
Please assign me these modules:
- dubbo-common-extensions
- dubbo-cross-thread-extensions
- dubbo-filter-seata

**Comment by songxiaosheng** [comment]
> Please assign me these modules:
> 
> * dubbo-common-extensions
> * dubbo-cross-thread-extensions
> * dubbo-filter-seata

done

**Comment by xixingya** [comment]
please asign me  dubbo-serialization-fastjson、dubbo-serialization-gson、dubbo-serialization-jackson @songxiaosheng

**Comment by songxiaosheng** [comment]
> please asign me dubbo-serialization-fastjson、dubbo-serialization-gson、dubbo-serialization-jackson @songxiaosheng

done

**Comment by robin977** [comment]
Please assign me dubbo-metadata-report-etcd

**Comment by songxiaosheng** [comment]
> Please assign me dubbo-metadata-report-etcd

done

**Comment by robin977** [comment]
Please assign me  dubbo-registry-nameservice

**Comment by songxiaosheng** [comment]
> Please assign me  dubbo-registry-nameservice
> @songxiaosheng

done
